libs/designsystem/src/lib/components/segmented-control/segmented-control.component.scss

```
@@ -21,32 +21,62 @@
     }
   }
 
+  &.default-mode {
+    ion-segment {
+      --background: #{utils.get-color('white')};
+
+      display: inline-flex; // The segmented control itself is block-level, but we don't want the background to stretch
+      width: auto; // The segmented control itself is block-level, but we don't want the background to stretch
+      overflow: visible; // Ensures the badge is not cut off
+      contain: unset; // Ensures the badge is not cut off
```

In general one of the only good use cases for using `unset` is when assigned to the `all` property. In all other cases I prefer to just assign the value that `unset` will result in. That would be either `initial` or `inherit` depending on the property.

In this case I would suggest assigning either `initial` or explicitly `none` (the initial value of `contain`)

```suggestion
      contain: initial; // Ensures the badge is not cut off
```

or 

```suggestion
      contain: none; // Ensures the badge is not cut off
```

https://developer.mozilla.org/en-US/docs/Web/CSS/unset
https://developer.mozilla.org/en-US/docs/Web/CSS/contain#formal_definition

---

I just realised this may be code you just moved - not something you wrote, right?

---

You are right, just moved it, but lets change it anyways now that we have a good description of the context here! I have set it to `none`.